keyboards/alectardy98/alpinenoon/keymaps/alec/keymap.c

```diff
@@ -0,0 +1,219 @@
+    SPIT,                   //Macro for ":smiley_spit_surprised:" emoji on discord
+    KING,                   //Macro for ":pepe_king_royal:" emoji on discord
+
+};
+    
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {             //Code for macros
+        switch(keycode) {
+            case TEST:
+            if (record->event.pressed) {
+                SEND_STRING("TEST");
+                }break;
+            case VDRT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI( SS_DOWN(X_LCTRL) SS_TAP(X_RIGHT) SS_UP(X_LCTRL) ));
+                }break;
+            case VDLT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI( SS_DOWN(X_LCTRL) SS_TAP(X_LEFT) SS_UP(X_LCTRL) ));
+                }break;
+            case VDUP:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI(SS_TAP(X_TAB)"t"));
+                }break;
+            case VDDN:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI("d"));
+                }break;
+            case EXPL:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI("e"));
+                }break;
+            case SNIP:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LGUI(SS_LSFT("s")));
+                }break;
+            case COPY:
+            if (record->event.pressed) {
+                SEND_STRING(SS_TAP(X_F2) SS_LCTRL("ac"));
+                }break;
+            case PAST:
+            if (record->event.pressed) {
+                SEND_STRING(SS_TAP(X_F2) SS_LCTRL("v"));
+                }break;
+            case TASK:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTRL(SS_DOWN(X_RSHIFT)SS_TAP(X_ESCAPE)SS_UP(X_RSHIFT)));       // Windows
+                SEND_STRING(SS_LCTRL(SS_DOWN(X_LALT)SS_TAP(X_ESCAPE)SS_UP(X_LALT)));           //Mac
+                }break;
+            case QUIT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LALT(SS_TAP(X_F4)));                                          //Windows
+                SEND_STRING(SS_LCTRL("w"));                                                   //Mac
+                }break;
+            case DZRO:
+            if (record->event.pressed) {
+                SEND_STRING("00");
+                }break;
+            case ATAB:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LALT(SS_TAP(X_TAB)));
+                }break;
+            case LPRC:
+            if (record->event.pressed) {
+                SEND_STRING("(");
+                }break;
+            case RPRC:
+            if (record->event.pressed) {
+                SEND_STRING(")");
+                }break;
+                //Discord Section
+                case THIS:
+                if (record->event.pressed) {
+                    SEND_STRING(":this:" SS_TAP(X_ENTER) );
```

Comment:
```suggestion
                    SEND_STRING(":this:\n" );
```
etc

Reply:
It's standard C string escaping.
https://docs.qmk.fm/#/feature_send_string
https://en.wikipedia.org/wiki/Escape_sequences_in_C

Reply:
I just removed the keymap, this was out of the scope of my ability for now